plugins/woocommerce/src/Internal/Admin/EmailPreview/EmailPreview.php

```diff
@@ -439,12 +439,13 @@ private function get_placeholders( $email_object ) {
 		/**
 		 * Placeholders for email preview.
 		 *
-		 * @param WC_Order $placeholders Placeholders for email subject.
+		 * @param array    $placeholders Placeholders for email subject.
 		 * @param string   $email_type The email type to preview.
+		 * @param WC_Order|WP_User $email_object The object to render email with.
```

Comment:
Now I'm thinking that this should probably be `mixes` (as well as in the method param). Technically, WC Core emails only use `WC_Order` or `WP_User`, but an extension can register email with any object, in which case, this type hint is misleading. What do you think about changing both to `mixed`?

Reply:
Agreed. Fixed in https://github.com/woocommerce/woocommerce/pull/57272/commits/4193dbe935c0ec7f18a508f8a18cd706cab70fdc